src/client/sandbox/cookie.js

```diff
@@ -129,12 +142,11 @@ export default class CookieSandbox extends SandboxBase {
 
         var setCookieMsg = {
             cmd:    COMMAND.setCookie,
-            cookie: value,
+            cookie: this.getCookie(),
             url:    document.location.href
         };
 
-        // NOTE: Meanwhile, synchronize cookies with the server cookie jar.
-        transport.queuedAsyncServiceMsg(setCookieMsg);
+        this.throttledCookieSync(setCookieMsg);
```

Comment:
it should be named `syncCookies`, `Sync` suffix means that functions has `sync` nature, but we need to synchronize cookies here, also we can go without `throttled`, it's just an implementation details.